src/Components/CoinsListItem/CoinsListItem.js

```diff
@@ -136,33 +154,51 @@ export class CoinsListItem extends Component {
             <PercentageBarContainer>
               <ValuesContainer>
                 <Value
-                  colors={percBarColors.left[index % percBarColors.left.length]}
+                  colors={
+                    percentageBarColors.left[
+                      index % percentageBarColors.left.length
+                    ]
+                  }
                 >
                   <BulletCircle
                     colors={
-                      percBarColors.left[index % percBarColors.left.length]
+                      percentageBarColors.left[
+                        index % percentageBarColors.left.length
+                      ]
```

Comment:
const color =  percentageBarColors.left[ index % percentageBarColors.left.length ] and then you can reuse it. DRY